dss-server/src/test/java/io/github/ztkmkoo/dss/server/rest/DssRestServerTest.java

```diff
@@ -171,4 +235,28 @@ private static File loadFromTestResources(String path) throws UnsupportedEncodin
         final ClassLoader classLoader = DssRestServerTest.class.getClassLoader();
         return new File(URLDecoder.decode(classLoader.getResource(path).getFile(),"UTF-8"));
     }
+
+    private static void startOnDssJsonRestServer(DssRestServer dssRestServer) {
+        startOnNewThread(() -> {
+            dssRestServer
+                    .addDssRestService(new TestJsonService("test_json", "/test/json", DssRestMethodType.POST));
+            try {
+                dssRestServer.start();
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+        });
+    }
+
+    private static HttpResponse sendPostJsonRequest(String uri, String sendJsonBody) throws IOException {
+        final HttpClient httpClient = HttpClientBuilder.create().build();
+        final HttpPost request = new HttpPost(uri);
+
+        final StringEntity params = new StringEntity(sendJsonBody);
+
+        request.addHeader("content-type", "application/json");
+        request.setEntity(params);
+
+        return httpClient.execute(request);
+    }
```

Comment:
this method is too.

It does seem better that parameter add request method type.

Reply:
agree